src/components/index.js

```diff
@@ -1,3 +1,4 @@
 import Instructions from './Instructions.component'
+import Counter from './Counter.component'
 
-export { Instructions }
+export { Instructions, Counter }
```

Comment:
I made PR #7 to get rid of this file structure from the actual bootstrap repo, but gotta wait to see what @vrunjeti has to say. If it gets merged, then this has to be updated